site/public/js/server.js

```diff
@@ -169,11 +223,19 @@ function loadTestcaseOutput(div_name, gradeable_id, who_id, index, version = '')
                 enableKeyToClick();
             },
             error: function(e) {
-                alert("Could not load diff, please refresh the page and try again.");
-                console.log(e);
-                displayAjaxError(e);
+                if(output_xhrs.has(orig_div_name) && output_xhrs.get(orig_div_name).readyState == 0){
+                    output_xhrs.delete(orig_div_name);
+                    CollapseTestcaseOutput(orig_div_name, index, loadingTools);
+                    console.log("JSON Load Aborted");
+                }
+                else
+                {
+                    alert("Could not load diff, please refresh the page and try again.");
```

Comment:
Same comment about brace style

Reply:
Opening curly brace has been moved to the end of the preceding line.